### ISSUE TITLE
stream functionality?

### ISSUE THREAD

**Issue by Miregen** [fudgeu/playlist#17]
I have been previously using this other mod called "Music Player" (there are two mods with the same name, it's not on Modrinth so I'll just give the link for you to see for yourself) and it has some online functionalities I'd like to see incorporated into this mod if it could even be possible and you're open to the suggestion (I don't know how hard having something work online would be, so it might not be a very good idea). 

I also find that it would be interesting to see if this might help with the custom-added music issue, possibly even being able to convert soundtracks directly into a resource pack like with the VanillaTweaksDownloader mod.

> (curseforge.com/minecraft/mc-mods/music-player) - by HyCraftHD

Thank you, kindly. (and sorry for the wall of text, lots of ideas came to mind when I saw this and I couldn't not have told someone about them)

**Comment by fudgeu** [comment]
Yup! Currently this mod is focused more on controlling the game's built in music, as that was its original purpose (a large inspiration of it being I really wanted music from some parts of the game to play in others, like say I wanted the music that mainly plays in creative or on the main menu to also play in survival mode). Custom music is something I had also planned, but more in the future once I have the rest of the framework of the mod up and going. The next update should bring *a lot* of new features that bring that framework closer to completion.
With the addition of custom music, I do want to make sure that I make it as easy as possible - including a 'stream' like feature where you can input a YouTube link and have it fetch the song. I'll do my best to implement it when the time comes (hopefully soon, if everything goes well)
Exporting playlists so that you can share with your friends is also a feature I had planned in mind, but again, further down the line, lol
Thank you!
